rom/dev/src/lock.rs

```diff
@@ -54,34 +52,8 @@ pub fn lock_registers(env: &mut RomEnv, reset_reason: ResetReason) {
 ///
 /// * `env` - ROM Environment
 #[cfg_attr(not(feature = "no-cfi"), cfi_mod_fn)]
-fn lock_cold_reset_reg(env: &mut RomEnv) {
-    // Lock the FMC TCI in data vault until next cold reset
-    env.data_vault
-        .lock_cold_reset_entry48(ColdResetEntry48::FmcTci);
-
-    // Lock the FMC SVN  in data vault until next cold reset
-    env.data_vault
-        .lock_cold_reset_entry4(ColdResetEntry4::FmcSvn);
-
-    // Lock the FMC entry point in data vault until next cold reset
-    env.data_vault
-        .lock_cold_reset_entry4(ColdResetEntry4::FmcEntryPoint);
-
-    // Lock the Owner Public Key Hash in data vault until next cold reset
-    env.data_vault
-        .lock_cold_reset_entry48(ColdResetEntry48::OwnerPubKeyHash);
-
-    // Lock the Ecc Vendor Public Key Index in data vault until next cold reset
-    env.data_vault
-        .lock_cold_reset_entry4(ColdResetEntry4::EccVendorPubKeyIndex);
-
-    // Lock the Lms Vendor Public Key Index in data vault until next cold reset
-    env.data_vault
-        .lock_cold_reset_entry4(ColdResetEntry4::PqcVendorPubKeyIndex);
-
-    // Lock Cold Reset Status register in data vault until next cold reset
-    env.data_vault
-        .lock_cold_reset_entry4(ColdResetEntry4::RomColdBootStatus);
+fn lock_cold_reset_reg(_env: &mut RomEnv) {
+    // [TODO][CAP2] Lock the cold reset entries via PMP.
```

Comment:
Maybe open an issue for this

Reply:
Added: https://github.com/chipsalliance/caliptra-sw/issues/1859